scripts/compose.py

```diff
@@ -522,13 +523,35 @@ def __init__(self, **options):
             if self.at_least_version("6.3"):
                 self.environment["XPACK_XPACK_MAIN_TELEMETRY_ENABLED"] = "false"
 
+    @classmethod
+    def add_arguments(cls, parser):
+        super(Kibana, cls).add_arguments(parser)
+        parser.add_argument(
+            "--kibana-with-canvas",
+            const="2174",
```

Comment:
can you add a comment here what this number signifies? I assume it is the patch-level version number

Also, looking at the interplay between `nargs="?"` and `const`, it will `--kibana-with-canvas` will result in `0.1.2174` being build, while `--kibana-with-canvas 1234` will build `0.1.1234`, right? Might make sense to explain that in `help`

Reply:
Sure, I'll expand the help.  I was just messing around with this over lunch and didn't expect anyone to actually look at it :) (thank you)

Reply:
Updated help and switched to accepting the entire version.